src/outfit.ts

```diff
@@ -50,8 +50,14 @@ export function baggoOutfit(includeFamiliar = true): Outfit {
   outfit.setModes({ parka: "ghostasaurus" });
 
   const valuator = SimulatedState.baseline(outfit).makeValuator();
+  let modifier = `${valuator(1, 0).toFixed(2)}familiar weight, ${valuator(0, 1).toFixed(
+    2
+  )}item drop`;
+  if (get("baggo_2crs")) {
+    modifier = `+familiar weight`;
```

Comment:
I think adding `2000 max` after the item drop would achieve what you are wanting. Need to add that regardless.